Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -261,3 +261,5 @@ Delete the cluster
 .. code-block:: shell-session
 
    openshift-install destroy cluster --dir="${CLUSTER_NAME}"
+
+To learn more about OpenShift and Cilium, checkout eBPF & Cilium Office Hours `episode 31: OpenShift Test Environment with Cilium <https://youtu.be/4jwIBS5IzYg?t=945>`__.
```

Comment:
![Screenshot 2022-07-14 at 12 07 20](https://user-images.githubusercontent.com/42554056/178969194-53512711-e8b8-43eb-952d-fbbfd58122fc.png)

I left this format in the Talk section of the `bpf.rst` so It would be clear as possible just like other talks listed there. 

Does that work in this cases? What do you think?